modules/generator/processor/servicegraphs/config.go

```diff
@@ -39,6 +39,9 @@ type Config struct {
 
 	// If enabled attribute value will be used for metric calculation
 	SpanMultiplierKey string `yaml:"span_multiplier_key"`
+
+	// EnableExtraUninstrumentedServicesLabels enables additional labels for uninstrumented services
+	EnableExtraUninstrumentedServicesLabels bool `yaml:"enable_extra_uninstrumented_services_labels"`
```

Comment:
similar thought to the below on the edge struct. i'm wondering if there's a way to safely encode this information in the `Dimensions` field above?

Reply:
Encoding this fully into `Dimensions` for this struct is more tricky.

On one side, `virtual_node` is not a resource attribute anywhere, so encoding that into `Dimensions` could be counter-intuitive.

Secondly, while the other four are resource attributes, encoding them here would mean they also need to enable the client/server prefixes for them.

Let me know what you think.

(docs updated too)